presto-main/src/main/java/com/facebook/presto/sql/planner/plan/WindowNode.java

```diff
@@ -236,8 +235,8 @@ public Frame(
                 @JsonProperty("startValue") Optional<Symbol> startValue,
                 @JsonProperty("endType") FrameBound.Type endType,
                 @JsonProperty("endValue") Optional<Symbol> endValue,
-                @JsonProperty("originalStartValue") Optional<Expression> originalStartValue,
-                @JsonProperty("originalEndValue") Optional<Expression> originalEndValue)
+                @JsonProperty("originalStartValue") Optional<String> originalStartValue,
```

Comment:
@highker James, I'm seeing that originalStartValue and originalEndValue correspond directly to startValue and endValue. Do we need these at all? Can we remove originalXxx and use start/endValue in the plan printer?

Reply:
Unfortunately we can't. Subplan translator (in `QueryPlanner`) maps each original expression to a symbol. `startValue` and `endValue` will only be auto-generated symbols. The expressions will be appended as a projection (check `outputTranslations` in `QueryPlanner`).